core/server/apps/subscribers/lib/helpers/subscribe_form.js

```diff
@@ -1,50 +1,41 @@
 // # Subscribe Form Helper
 // Usage: `{{subscribe_form}}`
-var _ = require('lodash'),
-
+const _ = require('lodash'),
     // (Less) dirty requires
     proxy = require('../../../../helpers/proxy'),
     templates = proxy.templates,
     urlService = proxy.urlService,
     SafeString = proxy.SafeString,
+    params = ['error', 'success', 'email'];
 
-    params = ['error', 'success', 'email'],
-
-    subscribeScript;
+    /**
+     * This helper script sets the referrer and current location if not existent.
+     *
+     * document.querySelector['.location']['value'] = document.querySelector('.location')['value'] || window.location.href;
+     */
+let subscribeScript =
+    '<script>(function(g,h,o,s,t){h[o](\'.location\')[s]=h[o](\'.location\')[s] || g.location.href;h[o](\'.referrer\')[s]=h[o](\'.referrer\')[s] || h.referrer;})(window,document,\'querySelector\',\'value\');</script>';
```

Comment:
This was much more readable before 🤔 What was the reason to change it to a one-liner?

Reply:
@kirrg001 It is a literal string value then assigning a value when declaration is more concise, if it is what you meant. On the other hand, `subscribeScript` value was already a long line before, I just repositioned it.

Reply:
I think something like

```
subscribeScript =
`<script>
    (function(g,h,o,s,t) {
    	h[o]('.location')[s]=h[o]('.location')[s] || g.location.href;
    	h[o]('.referrer')[s]=h[o]('.referrer')[s] || h.referrer;
    })(window,document,'querySelector','value');
</script>`;
```

Reply:
@vikaspotluri123 Done, thanks :)